sphinx/ext/autodoc/__init__.py

```diff
@@ -417,7 +417,7 @@ def import_object(self, raiseerror: bool = False) -> bool:
             try:
                 ret = import_object(self.modname, self.objpath, self.objtype,
                                     attrgetter=self.get_attr,
-                                    warningiserror=self.config.autodoc_warningiserror)
+                                    )
```

Comment:
I think you can remove the trailling , and put ) on the same line as 'self.get_attr'